cpu/stm32f4/include/periph_cpu.h

```diff
@@ -115,6 +140,13 @@ typedef struct {
 void gpio_init_af(gpio_t pin, gpio_af_t af);
 
 /**
+ * @brief   Configure the given pin to be used as ADC input
+ *
+ * @param[in] pin       pin to configure
+ */
+void gpio_init_analog(gpio_t pin);
```

Comment:
no, does not make sense. This is an internal, cpu-specific function and thus should not be part of the public GPIO interface